docs/bpmn2/example.bpmn

```diff
@@ -0,0 +1,182 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Remove

Reply:
That file has been deleted because it was no longer necessary